sdks/python/apache_beam/runners/portability/stager_test.py

```diff
@@ -167,6 +167,9 @@ def test_no_main_session(self):
 
   # xdist adds unpicklable modules to the main session.
   @pytest.mark.no_xdist
+  @pytest.mark.skip(
+      sys.platform == 'win32',
+      "Windows raise TypeError: can't pickle zipimport.zipimporter objects")
```

Comment:
Is there a bug in dill or cpython pickle that tracks this problem? Does this fail on all Python versions?

Reply:
Curious what happened to this skip. We're seeing flakes like this quite often BEAM-10987

Reply:
I see it this skip was not included in https://github.com/apache/beam/pull/12452.
As far i remember after some rebasings and further work on the tests it started to work and for many test runs it didn't fail so I removed this skip.

Reply:
Put up a PR to add it back in https://github.com/apache/beam/pull/12827